.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -8,40 +8,43 @@ assignees: ''
 ---
 
 **Describe the bug**
-A clear and concise description of what the bug is.
+<!-- A clear and concise description of what the bug is. -->
 
 ***Exception or Stack Trace***
-Add the exception log and stack trace if available
+<!-- Add the exception log and stack trace if available -->
 
 **To Reproduce**
-Steps to reproduce the behavior:
+<!-- Steps to reproduce the behavior: -->
 
 ***Code Snippet***
-Add the code snippet that causes the issue.
+<!-- Add the code snippet that causes the issue. -->
 
 **Expected behavior**
-A clear and concise description of what you expected to happen.
+<!-- A clear and concise description of what you expected to happen. -->
 
 **Screenshots**
-If applicable, add screenshots to help explain your problem.
+<!-- If applicable, add screenshots to help explain your problem. -->
 
 **Setup (please complete the following information):**
- - OS: [e.g. iOS]
- - IDE: [e.g. IntelliJ]
- - Library/Libraries: [e.g. com.azure.spring:spring-cloud-azure-starter-keyvault-secrets:4.0.0 (groupId:artifactId:version)]
- - Java version: [e.g. 8] 
- - App Server/Environment: [e.g. Tomcat, WildFly, Azure Function, Apache Spark, Databricks, IDE plugin or anything special]
- - Frameworks: [e.g. Spring Boot, Micronaut, Quarkus, etc] 
-
+- Github link with tag or branch: <!-- [e.g. https://github.com/Azure-Samples/azure-spring-boot-samples/tree/spring-cloud-azure_v4.0.0/cosmos/azure-spring-data-cosmos/cosmos-multi-database-single-account] -->
+- OS: <!-- [e.g. iOS] -->
+- IDE: <!-- [e.g. IntelliJ] -->
+- Library/Libraries: <!-- [e.g. com.azure.spring:spring-cloud-azure-starter-keyvault-secrets:4.0.0 (groupId:artifactId:version)] -->
```

Comment:
how about putting sample and library as close as possible:
- OS
- IDE
- Sample Path
- Library/Libraries

Reply:
I should have updated this order.